<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGisDemo</class>
 <widget class="QWidget" name="QGisDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGisDemo</string>
  </property>
  <widget class="QgsGraphMap" name="qgsMap" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1200</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="select_combox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>选中模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>直线模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>虚拟墙模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{


}
QPushButton::checked
{
	#background-color:rgba(255,255,0,255);

}</string>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="all_clear_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{


}
QPushButton::checked
{
	#background-color:rgba(255,255,0,255);

}</string>
   </property>
   <property name="text">
    <string>all_clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_attribute_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{


}
QPushButton::checked
{
	#background-color:rgba(255,255,0,255);

}</string>
   </property>
   <property name="text">
    <string>保存字段</string>
   </property>
  </widget>
  <widget class="QPushButton" name="min_path">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>最短路径</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsGraphMap</class>
   <extends>QWidget</extends>
   <header>qgsgraphmap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
